data/brands/shop/supermarket.json

```diff
@@ -432,6 +432,21 @@
         "shop": "supermarket"
       }
     },
+    {
+      "displayName": "Bhat-Bhateni Super Market",
+      "id": "bhatbhateni-utwl8d",
+      "locationSet": {"include": ["np"]},
+      "matchNames": ["Bhatbhateni Super Market"],
+      "tags": {
+        "brand": "Bhat-Bhateni Super Market",
+        "brand:wikidata": "Q104847792",
+        "brand:wikipedia": "en:Bhat-Bhateni Super Market",
+        "brand:website": "https://www.bbsm.com.np/",
```

Comment:
```suggestion
```
Please note that we don't support `website` tags in NSI as these can be found on the entry's respective Wikidata item, see [this wiki article](https://github.com/osmlab/name-suggestion-index/wiki/Contributing#tags-required). If the Wikidata item doesn't already have an `official website` statement, you read more about how you can add it [here](https://github.com/osmlab/name-suggestion-index/wiki/Editing-Wikidata)

We [recently removed](https://github.com/osmlab/name-suggestion-index/commit/688cf6af6cf80e0d3bb511d28648652d5151c896) `:wikipedia` tags on NSI with this same reasoning.